src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/TargetFrameworkMoniker.cs

```diff
@@ -16,8 +16,8 @@ public record TargetFrameworkMoniker(string Name)
 
         public bool IsNetStandard => Name.StartsWith(NetStandardNamePrefix, StringComparison.OrdinalIgnoreCase);
 
-        public bool IsNetCore => Name.StartsWith(NetPrefix, StringComparison.OrdinalIgnoreCase) && Name.Contains('.');
+        public bool IsNetCore => Name.StartsWith(NetPrefix, StringComparison.OrdinalIgnoreCase) && Name.Contains(".");
 
-        public bool IsWindows => Name.Contains("windows", StringComparison.OrdinalIgnoreCase);
+        public bool IsWindows => Name.Contains("windows");
```

Comment:
Why not be explicit about the StringComparison we want?

Reply:
Because that API is not available before .NET 5

Reply:
Ah, didn't realize that one was new.